`fix`: AccountTransactions `int64` props change to `strings`

ivaylogarnev-limechain opened the pull request:
**Description**:
This PR converts account transaction number properties to strings and introduces enums for better organization.

**Related issue(s)**:
 #284

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

rwalworth commented on docs/test-specifications/crypto-service/AccountCreateTransaction.md:
```diff
@@ -33,13 +33,13 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
+| autoRenewPeriod           | string                                            | optional          | Units of seconds                                                                                                                            |
 | memo                      | string                                           | optional          |                                                                                                                                             |
-| maxAutoTokenAssociations  | int32                                            | optional          |                                                                                                                                             |
+| maxAutoTokenAssociations  | string                                            | optional          |                                                                                                                                             |
```
Do we need to change `int32` types as well?

ivaylogarnev-limechain replied:
Not needed.

rwalworth commented on src/utils/enums/error-codes.ts:
```diff
@@ -0,0 +1,4 @@
+export enum ErrorStatusCodes {
+  ACCOUNT_DELETED = 72,
+  INTERNAL_ERROR = -32603,
+}
```
So there's `ErrorStatusCodes` and also `ERROR_STATUS_CODES`? Could these be combined in any way? Or is one of them unnecessary?

ivaylogarnev-limechain replied:
Removed the `ERROR_STATUS_CODES`

rwalworth commented on src/tests/token-service/test-token-create-transaction.ts:
```diff
@@ -1711,7 +1711,7 @@ describe("TokenCreateTransaction", function () {
   });
 
-  describe("Expiration Time", () => {
+  describe.only("Expiration Time", () => {
```
Intentional change?

ivaylogarnev-limechain replied:
Sometimes I forget to remove those, which is like instructing the system to run only that specific test. I should probably implement Nathan's suggestion instead of Husky. This way, when I attempt to commit, it will prompt me to address these unnecessary items.

rwalworth reviewed (commented):
Having trouble running these tests individually by themselves.. when I try to run `npm test src/tests/crypto-service/test-account-create-transaction.ts` it still runs everything

rwalworth reviewed (approved):
Works with C++ server

ivaylogarnev-limechain commented:
> Having trouble running these tests individually by themselves.. when I try to run `npm test src/tests/crypto-service/test-account-create-transaction.ts` it still runs everything

I have modified the package.json file and added an `npm run test:file `command. This new command allows to specify a particular file for testing, while the default `npm run test` command continues to run all tests. README updated too.

ivaylogarnev-limechain commented:
Works on  JS & GO servers
